config/routes.rb

```diff
@@ -144,6 +144,13 @@
               get ":month/:year", to: "projects#show", constraints: {month: MONTH_1_12_REGEX, year: YEAR_2000_2499_REGEX}, as: :show
               get ":month/:year/csv", to: "projects#csv", constraints: {month: MONTH_1_12_REGEX, year: YEAR_2000_2499_REGEX}, as: :csv
             end
+            namespace :grant_management_and_finance_unit, path: "grant-management-and-finance-unit" do
+              namespace :conversions do
+                get "/", to: "projects#index"
+                get ":month/:year", to: "projects#show", constraints: {month: MONTH_1_12_REGEX, year: YEAR_2000_2499_REGEX}, as: :show
+                get ":month/:year/csv", to: "projects#csv", constraints: {month: MONTH_1_12_REGEX, year: YEAR_2000_2499_REGEX}, as: :csv
```

Comment:
Personally I would do this as a `format.csv` content-negotiation response, rather than a separate action, but it's not a big enough deal to worry about. Happy to discuss though if anyone's interested 😁